pkg/wasm/runtime/wasmer/wasmer_unsupported.go

```diff
@@ -0,0 +1,21 @@
+//go:build !wasmer
```

Comment:
I had to add an empty file here because there are places in mosn which do an underscore import of wasmer. This allows those call sites to stick around even if the build flag isn't there. cc @taoyuanyuan